src/map/entities/mobentity.cpp

```diff
@@ -732,6 +727,12 @@ void CMobEntity::OnMobSkillFinished(CMobSkillState& state, action_t& action)
         }
         PTarget->StatusEffectContainer->DelStatusEffectsByFlag(EFFECTFLAG_DETECTABLE);
     }
+    PTarget = static_cast<CBattleEntity*>(state.GetTarget());
+    if (PTarget->isDead())
+    {
+        battleutils::ClaimMob(PTarget, this);
```

Comment:
I realize the secondary targets shouldn't, but shouldn't the primary target of the AoE always get claimed?

(Same for magic too)

Reply:
I believe I read that mobskills don't actually claim, only normal attacks/magic, but I may be wrong about that. Magic should be claiming the primary target.